internal/installercache/installercache.go

```diff
@@ -161,14 +168,16 @@ func (i *Installers) getDiskUsageIncludingHardlinks() (uint64, error) {
 	return usedBytes, nil
 }
 
-func (i *Installers) extractReleaseIfNeeded(path, releaseID, releaseIDMirror, pullSecret, ocpVersion string, ocRelease oc.Release) (extractDuration float64, cached bool, err error) {
+func (i *Installers) extractReleaseIfNeeded(path, releaseID, releaseIDMirror, pullSecret, ocpVersion string, ocRelease oc.Release, majorMinorVersion string) (extractDuration float64, cached bool, err error) {
 	_, err = os.Stat(path)
 	if err == nil {
+		i.metricsAPI.InstallerCacheGetReleaseCached(majorMinorVersion, true)
 		return 0, true, nil // release was found in the cache
 	}
 	if !os.IsNotExist(err) {
 		return 0, false, err
 	}
+	i.metricsAPI.InstallerCacheGetReleaseCached(majorMinorVersion, false)
```

Comment:
IIUC this could be done with one call in the main Get function:
```
			release, err := i.get(releaseID, releaseIDMirror, pullSecret, ocRelease, ocpVersion, clusterID, majorMinorVersion)
			if err == nil {
				return release, nil
			}
                        i.metricsAPI.InstallerCacheGetReleaseCached(majorMinorVersion, release.cached)
                        _, isCapacityError := err.(*errorInsufficientCacheCapacity)
			if !isCapacityError {
				return nil, errors.Wrapf(err, "failed to get installer path for release %s", releaseID)
			}
time.Sleep(i.config.ReleaseFetchRetryInterval)
```

This will avoid passing down the chain major/minor version and have one single point for this metric